src/main/java/frc/robot/Constants.java

```diff
@@ -71,6 +76,18 @@ public static class Drive
         public static final Rotation2d MODULE_BR_OFFSET   = Rotation2d.fromRadians(-1.63);
     }
 
+    public static class ShooterBed // FIXME: Update all these values
+    {
+        public static final Rotation2d BED_ANGLE_OFFSET = Rotation2d.fromRadians(0);
+        public static final Rotation2d MAX_BED_ANGLE    = Rotation2d.fromRadians(Math.PI / 2);
+        public static final Rotation2d MIN_BED_ANGLE    = Rotation2d.fromRadians(0);
```

Comment:
These can stay for now, but we'll eventually want to specify minimum and maximum angles in degrees. I don't think in radians 😅